models/theme.js

```diff
@@ -0,0 +1,36 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var connection = mongoose.createConnection("mongodb://localhost/niji");
+
+var themeSchema = new Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  description: {
+    type: String,
+    required: true
+  },
+  target: {
```

Comment:
Typical usage of `zenbu` has colors in one theme, which are then applied across all terminals. As of right now, the only target is "colors", since that's the only convention we currently have.